CIP-????/README.md

```diff
@@ -0,0 +1,530 @@
+---
+CIP: ????
```

Comment:
```suggestion
CIP: 130
```
Please also retitle containing directory to `CIP-0130` and update the rendered proposal link in your OP 🎉